ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-virtual-topologies.md

```diff
@@ -76,6 +88,17 @@
 
         # Set DSCP for matched traffic.
         dscp: <int; 0-63>
+        constraints:
+
+          # Jitter requirement for this load balance policy in milliseconds.
+          jitter: <int; 0-10000>
+
+          # One way delay requirement for this load balance policy in milliseconds.
+          latency: <int; 0-10000>
+
+          # Loss Rate requirement in percentage for this load balance policy.
+          # Value between 0.00 and 100.00 %
```

Comment:
I know it is $ref but I don't like the last `%` here since people may try to add it to the value (since it is a string).
```suggestion
          # Value between 0.00 and 100.00
```